data/systemd/Makefile.am

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+systemdsystemunitdir = $(SYSTEMD_SYSTEM_UNIT_DIR)
+
+SYSTEMD_UNITS_GENERATED := \
+	snapd.autoimport.service.in	\
+	snapd.core-fixup.service.in \
+	snapd.refresh.service.in \
+	snapd.service.in \
+	snapd.snap-repair.service.in \
+	snapd.system-shutdown.service.in
+
+SYSTEMD_UNITS = \
+	$(SYSTEMD_UNITS_GENERATED:.in=) \
+	$(wildcard *.timer) \
```

Comment:
Could you please list them manually here, it's not a big thing but I like to be explicit about this.